all/projectAlcazar/src/sys/gpio.c

```diff
@@ -12,7 +12,7 @@ void setPinMode(volatile GPIO_Type * port, unsigned int pinNumber, ePinMode pinM
 {
 	if (!port || pinNumber > 15)
 		return;
-
+	port->MODER &= ~( 0x3 << 2*pinNumber );// Deletes previous setting of the pin mode
```

Comment:
Please modify to use MODERPOS macro. When you modify commit and push yopur changes, it will show automatically to this PR, no need to open another PR

Reply:
No need to do above modification, I will do it later, merged your PR :)